x_xy/io/xml/abstract.py

```diff
@@ -111,49 +111,33 @@ def to_xml(element: T, pos_min: jax.Array, pos_max: jax.Array):
         element.set("pos_max", _to_str(pos_max))
 
 
-def _from_xml_vispy(attr: ATTR):
-    """Find all keys starting with `vispy_`, and return subdict without that prefix.
-    Also convert all arrays back to list[float], because of `struct.field(False)`.
-    Otherwise jitted functions with `sys` input will error on second execution, since
-    it can't compare the two vispy_color arrays.
-    """
+def _to_xml_colors(element: T, geom: base.Geometry) -> None:
+    "Copy pasted from you"
```

Comment:
Better docstring, or simply ommit it's pretty trivial what it does

Reply:
merged into `_to_xml_geom_processing`